packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -204,6 +208,34 @@ function checkResponseFormat(fileName, actualReponse, expectedResponse) {
   return missingKeys;
 }
 
+function findSchema(file) {
+  const execApisOpenRpcData = require('../../../../openrpc_exec_apis.json');
```

Comment:
QQ: Why do we need to import this json file every time this function is called? Can't we just import it only once at the top and then reuse it here?

Reply:
I did it in this way, because mocha loads all files and their imports even when we run the mocha command with the -g option (for a specific file). Therefore, it tries to load this file in the other workflows and the file still does not exist, it is created only in the conformity workflow. I moved it in the beginning of each test in the "it", so at least it loads for every directory and not every file. Another option is to somehow exclude this test file when other workflos are executed.

Reply:
Okay, I see now. Yes, for every directory would be slightly better. Let's leave it like that.